test/test_vmap.py

```diff
@@ -0,0 +1,113 @@
+from torch.testing._internal.common_utils import TestCase, run_tests
+import torch
+from torch import vmap
```

Comment:
Do we have plans for a more structured testing strategy? In principle you have to test vmap for *every* operator and it would be much better if we can incorporate vmap testing as part of, e.g., `common_methods`

Reply:
My currently testing strategy for operators is to construct a new table for vmap edge cases for every operator. `common_method_invocations` doesn't contain enough information for vmap tests.

I've considered extending `common_method_invocations` but that idea did not go very far